recipes/pymatgen/meta.yaml

```diff
@@ -0,0 +1,139 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pymatgen" %}
+{% set version = "4.5.2" %}
+{% set sha256 = "d5cf9ed832972c9c0de2cc1d3e2fff0ed71d8a8ad6f538118dd0680d933f9618" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pmg = pymatgen.cli.pmg:main
+    - feff_input_generation = pymatgen.cli.feff_input_generation:main
+    - feff_plot_cross_section = pymatgen.cli.feff_plot_cross_section:main
+    - feff_plot_dos = pymatgen.cli.feff_plot_dos:main
+    - gaussian_analyzer = pymatgen.cli.gaussian_analyzer:main
+    - get_environment = pymatgen.cli.get_environment:main
+    - pydii = pymatgen.cli.pydii:main
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9
+    - six
+    - requests
+    - pyyaml >=3.11
+    - monty >=0.9.6
+    - scipy >=0.14
+    - pydispatcher >=2.0.5
+    - tabulate
+    - spglib >=1.9.8.7
+    - matplotlib >=1.5
+    - palettable >=2.1.1
+    - enum34  # [py27]
+    - toolchain
```

Comment:
Are these all needed at build time? If not, please drop them. Probably only need `python`, `setuptools`, and `toolchain`.